lib/findExports.js

```diff
@@ -404,6 +404,11 @@ function findRootNodes(ast) {
   return realRootNodes;
 }
 
+function findUnnamedExports(data) {
+  const regex = /customElements\.define\s*\(\s*['"]([0-9a-z-]+)['"]\s*,\s*[a-zA-Z0-9_$]+\s*\)/g
+  return Array.from(data.matchAll(regex)).map(t => `<${t[1]}>`)
+}
+
```

Comment:
The regex here is fine, but make sure to add tests that show the behavior in different edge-cases so that they're clearly documented (like the commented-out code case you mentioned).

Reply:
I gave this some more thought and realized the problems don't end at commented code. It would also pick up the pattern inside of strings etc.

I'm becoming more certain that traversing the AST tree is the proper way to go. Do you have any indication on how much work that would involve? Guess it couldn't be too difficult?

Reply:
Hard to say, these things tend to be very explorative in nature. I usually TDD things like this, start with adding a test case to `findExports-test.js` and iterate from there. 

The AST traversing helpers in this library aren't super intuitive, and there might be better alternatives out there. But with a TDD approach things tend to work quite well.